packages/expo-secure-store/android/src/main/java/expo/modules/securestore/SecureStoreModule.java

```diff
@@ -494,7 +497,7 @@ public String getKeyStoreAlias(ReadableArguments options) {
           .setEndDate(new Date(Long.MAX_VALUE))
           .build();
 
-      KeyPairGenerator keyPairGenerator = KeyPairGenerator.getInstance(KeyProperties.KEY_ALGORITHM_RSA, keyStore.getProvider());
```

Comment:
I prefer to use the constants -- is this causing an Android API compatibility issue?

Reply:
I prefer to use the constants too, however it triggered the `InlinedApi` lint warning:

> This check scans through all the Android API field references in the application and flags certain constants, such as static final integers and Strings, which were introduced in later versions. *These will actually be copied into the class files rather than being referenced, which means that the value is available even when running on older devices.* In some cases that's fine, and in other cases it can result in a runtime crash or incorrect behavior. It depends on the context, so consider the code carefully and decide whether it's safe and can be suppressed or whether the code needs to be guarded.

Basing on the bolded text I have suppressed the warnings on the offending lines.